src/Modules/Backend/Domain/User/Command/CreateUser.php

```diff
@@ -7,6 +7,10 @@
 
 final class CreateUser extends UserDataTransferObject
 {
+    public bool $enableTwoFactorAuthentication = false;
+
+    public ?string $qrCode = null;
```

Comment:
just wondering, do we actually ever need this while adding a user?
While adding a user I don't think you'll be adding your own user, it will be the admin that creates the user, so they should just configure it when they log in the first time
@tijsverkoyen what do you think?

Reply:
Correct. Only users themself should be able to setup 2FA

Reply:
fixed 0800d541db23b391af25d1d4d12fa80efc4a3c59